packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx

```diff
@@ -70,7 +70,13 @@ const AddWalletButton = ({ device, instances, addDeviceInstance, selectDeviceIns
                               })
                     }
                 >
-                    <Translation id="TR_ADD_WALLET" />
+                    {/* check if users has standard wallet enabled 
+                     (device.useEmptyPassphrase === true) means that standard wallet is enabled */}
+                    {device.useEmptyPassphrase ? (
+                        <Translation id="TR_ADD_HIDDEN_WALLET" />
```

Comment:
```suggestion
                        <Translation id={device.useEmptyPassphrase ? 'TR_ADD_HIDDEN_WALLET' : 'TR_ADD_WALLET'} />
```

Reply:
on second thought i dont think checking `device` is enough, `device` is a currently used device right? so it might have `useEmptyPassphrase: false` but it doesn't mean that you already have "standard wallet" but its not currently selected
i think `instances.find(d => d.useEmptyPassphrase)` would be better

try it out